woocommerce/woocommerce
# Add empty state when no attributes

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds an empty state when there are no attributes.

![Screenshot 2023-11-27 at 11 51 11](https://github.com/woocommerce/woocommerce/assets/1314156/08145f91-06c8-4849-8591-ce120be1f9bf)

Closes #39641.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a WooCommerce store with this branch and enable the product editor using **WooCommerce > Settings > Advanced > Features**
2. Go to **Products > Add New > Variations**
3. Press `Add new` under `Variation options`
4. If there aren't attributes, you'll see the message `Nothing yet. Type to create.`
5. Create an attribute and press `+ Add another`
6. Verify that the option `Nothing yet. Type to create.` is visible again
7. Go to the **Organization** tab and press the `Add new` button in the `Attributes` section.
8. Verify that the message `Nothing yet. Type to create.` is visible in the selector when there aren't attributes created



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
